README.rst

```diff
@@ -334,6 +334,12 @@ Example of usage:
     $ ARGS='--sqlite-test-db-uri sqlite+pysqlite:///test_sqlalchemy_filters.db' make coverage
 
 
+Python 2
+--------
+
+There is no active support for python 2, however it is compatiable as of January 2019, if you install funcsigs.
```

Comment:
We could replace `January 2019` with the next version number (or at least add the version as well) when we cut the next release.

For now, we could replace `January` with `February` since this hasn't landed yet on `master`, even though the PR was created last month.